README.md

```diff
@@ -150,6 +150,7 @@ Validator                               | Description
 **isSurrogatePair(str)**                | check if the string contains any surrogate pairs chars.
 **isUppercase(str)**                    | check if the string is uppercase.
 **isSlug**                              | Check if the string is of type slug. `Options` allow a single hyphen between string. e.g. [`cn-cn`, `cn-c-c`]
+**isStrongPassword(str, requirementOptions?, scoringOptions?)**   | Check if a password is strong or not. Allows for custom requirements or scoring rules. If `returnScore` is true, then the function returns an integer score for the password rather than a boolean.<br/>Default options: <br/>`{ minLength: 8, minLowercase: 1, minUppercase: 1, minNumbers: 1, minSymbols: 1, returnScore: false, pointsPerUnique: 1, pointsPerRepeat: 0.5, pointsForContainingLower: 10, pointsForContainingUpper: 10, pointsForContainingNumber: 10, pointsForContainingSymbol: 10 }`
```

Comment:
From my look of the eye, I thought, I needed to pass 3 parameters, though 2 are optional. 
But, from function arguments call, it takes `str & option`. 
I expected something like below...
```
isStrongPassword(str [, options])
```
It is not a priority but my suggestion for consistency and easy readability/usage.

Reply:
Missed this, thanks!